azerothcore/azerothcore-wotlk
# [Zone] Tanaris

Tanaris is now fully updated, since some creatures in zone was missing SmartAI.

Working on every zone at the moment, and will post updates once i'm done with each.

To test it:
Apply SQL
Restart your server

You can manually spawn NPC by ID to check SmartAI or you can go to zone and check like that as well.

.npc add ID
or
.tele Tanaris and run / test around.
